src/app/src/components/MapAndSidebar.jsx

```diff
@@ -1,123 +1,31 @@
-                                                />
-                                            }
-                                        >
-                                            <Route
-                                                component={
-                                                    VectorTileFacilitiesMap
-                                                }
-                                            />
-                                        </FeatureFlag>
-                                    )}
-                                />
-                                <Route
-                                    exact
-                                    path={facilitiesRoute}
-                                    render={() => (
-                                        <FeatureFlag
-                                            flag={VECTOR_TILE}
-                                            alternative={
-                                                <Route
-                                                    component={FacilitiesMap}
-                                                />
-                                            }
-                                        >
-                                            <Route
-                                                component={
-                                                    VectorTileFacilitiesMap
-                                                }
-                                            />
-                                        </FeatureFlag>
-                                    )}
-                                />
-                                <Route
-                                    exact
-                                    path={mainRoute}
-                                    render={() => (
-                                        <FeatureFlag
-                                            flag={VECTOR_TILE}
-                                            alternative={
-                                                <Route
-                                                    component={FacilitiesMap}
-                                                />
-                                            }
-                                        >
-                                            <Route
-                                                component={
-                                                    VectorTileFacilitiesMap
-                                                }
-                                            />
-                                        </FeatureFlag>
-                                    )}
-                                />
-                            </Switch>
-                        )}
-                        {hasError && <FacilitiesMapErrorMessage />}
-                    </Grid>
-                </Grid>
-            </Fragment>
-        );
-    }
+function mapStateToProps({ ui: { activeFilterSidebarTab } }) {
```

Comment:
Looks like this prop isn't being used, so this mapStateToProps can probably be removed